docs/docs-graphql/schema.graphqls

```diff
@@ -3426,6 +3433,9 @@ type LLM {
   """return the LLM's current environment"""
   env: Env!
 
+  """Indicates that the LLM can be synced or stepped"""
+  hasPrompt: Boolean!
```

Comment:
Walking without context: I don't understand what `hasPrompt` does exactly, why it's called that, or the relationship with the description.